### ISSUE TITLE
Every answer accepted as correct

### ISSUE THREAD

**Issue by stcwp** [rjz/jq-tutorial#3]
Example:

``` bash
👉  ./jq-tutorial pick
Pick
========================================

### Pick fields from an object

`jq` retrieves named properties from objects by using `.` syntax:

    $ echo '{"foo": { "bar": "a value" }}' | jq .foo

Nested values are accessible as well:

    $ echo '{"foo": { "bar": "a value" }}' | jq .foo.bar

### Pick elements from an array:

Elements in an array may be extracted by index:

    $ echo '["snap","crackle","pop"]' | jq .[1]

More than one index? No problem!

    $ echo '["snap","crackle","pop"]' | jq .[1, 2]

We can even extract *all* elements at once by omitting the indices:

    $ echo '["snap","crackle","pop"]' | jq .[]

type "data?" to see dataset or "help?" for more options

--------------------------------

Given:    'product' (type "data?" to view)
Challenge: Select the entire item (hint: don't overthink this!):

> data?
{
  "name": "Table",
  "price": 270.00
}


> .


--------------------------------

Given:    'product' (type "data?" to view)
Challenge: Select the item's price:

> efhwo


--------------------------------

Given:    'products' (type "data?" to view)
Challenge: Select all items from the list

> owhefu


--------------------------------

Given:    'products' (type "data?" to view)
Challenge: Select the second and third items in the list:

> fweoih


--------------------------------

Given:    'products' (type "data?" to view)
Challenge: Select the name of the first item in the list:

> fweohiu
★ "pick" completed with a gold star!
```

**Comment by rjz** [comment]
Well, that won't do! Thanks for the report, @stcwp.

I'll take a look at this soon (but would also be happy to bring in a PR, if you're so inclined).

**Comment by rjz** [comment]
Fixed by #4.
